jobs/kola-azure.Jenkinsfile

```diff
@@ -0,0 +1,179 @@
+def pipeutils, streams, official
+node {
+    checkout scm
+    pipeutils = load("utils.groovy")
+    streams = load("streams.groovy")
+    def pipecfg = pipeutils.load_config()
+    azure_testing_resource_group = pipecfg['azure-testing-resource-group']
+    azure_testing_storage_account = pipecfg['azure-testing-storage-account']
+    azure_testing_storage_container = pipecfg['azure-testing-storage-container']
```

Comment:
For consistency, we should add all of these as `def`s above.

Reply:
fixed